src/server/common/helpers/fetch/fetcher.js

```diff
@@ -12,13 +13,18 @@ import { getTraceId } from '~/src/server/common/helpers/tracing/tracing.js'
  */
 async function fetcher(url, options = {}) {
   const logger = createLogger()
+  const tracingHeader = config.get('tracing.header')
+  const xCdpRequestId = getXCdpRequestId()
```

Comment:
Should we call this traceId? and getTraceId() - we're making the header name configurable